### DISCUSSION TITLE
Choosing which pasteboard format when working with "paste"?

### DISCUSSION THREAD

**Discussion by spasm** [johnlindquist/kit#981] [Q&A]
Is it possible to choose which pasteboard format to "paste" in a script?

For example, I'm trying to get to the `text/html` representation of what's sitting on the pasteboard, however when using `paste()` I just get back the `text/plain` format.  In this particular example (e.g. copied from Tana) I can see there are three different representations of the copied content:

<img width="247" alt="image" src="https://user-images.githubusercontent.com/10431596/208316434-8e4d9dc9-09dc-4444-9615-5f34d7aa679c.png">

Is it possible to directly work with the `public.html` format within ScriptKit or do I need to do this processing outside?

Thanks!

**Comment by johnlindquist** [comment]
Not currently without a workaround. I'll make it possible in the next build with:

`await clipboard.readHTML()`
`await clipboard.readRTF()`
etc

The next build will be out before the New Year, just working around holidays and such.

**Reply by spasm** [reply]
that would be awesome, Thanks John!

**Reply by spasm** [reply]
Actually it looks like it's already out there and ready for use.  Thanks for the quick turnaround, I'll give it a test!

**Reply by johnlindquist** [reply]
@spasm Let me know if it works as expected. It was pretty straight forward to implement, but I'm sure I missed some edge cases.
